misk/src/test/kotlin/misk/web/actions/DashboardTabTest.kt

```diff
@@ -1,67 +1,67 @@
 package misk.web.actions
 
-import misk.web.DashboardTab
+import misk.web.DashboardTabProvider
 import org.junit.jupiter.api.Test
 import kotlin.test.assertFailsWith
 
 internal class DashboardTabTest {
```

Comment:
gonna rework this a bit, looks like it's failing now

Reply:
Thanks for fixing this!